urdfpy/utils.py

```diff
@@ -199,6 +218,7 @@ def get_filename(base_path, file_path, makedirs=False):
         The resolved filepath -- just the normal ``file_path`` if it was an
         absolute path, otherwise that path joined to ``base_path``.
     """
+    file_path = resolve_filepath(base_path, file_path)
```

Comment:
with comment above, I think you need a check to see if `resolve_filepath` returns `False`